js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+const SENTENCE_SUFFIX = '.';
+const SINGLE_WORD_SUFFIX = '...';
+const SINGLE_WORD_LIMIT_FOR_ABBREV = 15;
```

Comment:
What's this? Why not take this as parameter?

Reply:
I was following the Read.md so I didn't think of that but in general, it should be a parameter. I'll make this change. SINGLE_WORD_LIMIT_FOR_ABBREV should be parameter but why suffix should be a parameter?

Reply:
@unikuam Because what if tomorrow if I want the suffix to be `!!!` instead of 3 dots? If you take that as a parameter, dev can pass any value and get desired behavior. Also, you can make `...` as the default value. So the uses who don't pass a value of the suffix still get the correct behavior.  

Your function has not become much powerful because you can customize more things than you were able to do before.

Reply:
Thanks @ArfatSalman , yes, the function should be more powerful!! I just want to mention one thing here that ```...``` part was for the case if there is a long single word and if there is a sentence then it will append ```.```` to the first letter of last word. That is why I added two separate in the beginning so basically I have to take two parameters there one for single word & one for sentence because ```.``` does not make sense for a single word and ```..``` does not make sense for cases like  ```Anshul M.``` Kindly correct if I am wrong.